src/state/reducers/index.js

```diff
@@ -1,4 +1,11 @@
-// import all of your reducers into this file, and export them back out. 
+// import all of your reducers into this file, and export them back out.
 // This allows for the simplification of flow when importing reducers into your actions throughout your app.
+// import { combineReducers } from 'redux';
```

Comment:
Can delete this line of commented code.

Reply:
I'm actually utilizing that code in my next PR

Reply:
Ah, got it!